src/pps/pps.proto

```diff
@@ -711,21 +711,28 @@ message CreatePipelineRequest {
   string salt = 22;
   int64 datum_tries = 23;
   SchedulingSpec scheduling_spec = 24;
-  string pod_spec = 25; // deprecated, use pod_patch below
-  string pod_patch = 26; // a json patch will be applied to the pipeline's pod_spec before it's created;
+  string pod_spec = 25;; // deprecated, use pod_patch below
+  string pod_patch = 26;; // a json patch will be applied to the pipeline's pod_spec before its created;
```

Comment:
`it's` ("it is") was correct.  don't think 2 semicolons are necessary at the end of line here.